src/openrct2/ride/Ride.cpp

```diff
@@ -3586,16 +3611,17 @@ void Ride::MoveTrainsToBlockBrakes(TrackElement* firstBlock)
 static bool ride_initialise_cable_lift_track(Ride* ride, bool isApplying)
 {
     CoordsXYZ location;
-    for (StationIndex stationIndex = 0; stationIndex < OpenRCT2::Limits::MaxStationsPerRide; stationIndex++)
+    for (const auto& station : ride->GetStations())
     {
-        location = ride->stations[stationIndex].GetStart();
+        location = station.GetStart();
         if (!location.IsNull())
             break;
-        if (stationIndex == (OpenRCT2::Limits::MaxStationsPerRide - 1))
-        {
-            gGameCommandErrorText = STR_CABLE_LIFT_HILL_MUST_START_IMMEDIATELY_AFTER_STATION;
-            return false;
-        }
+    }
+
+    if (location.IsNull())
```

Comment:
This is not null by default.

Reply:
Good catch. In practice, this would never be hit, since a ride couldn't be tested without a station in the first place, but it's safe now anyway.